content/blocks/multi-block/crafting/steelworks-standard.hjson

```diff
@@ -1,7 +1,7 @@
 {
   type: GenericSmelter
   name: Standard steelworks
-  description: Multi-block factory.Provides more steel.\n Needs to enter water.\n Needs to provide refined carbon and refined iron.\n This is the [red]first [orange]multi-block factory[]of THC Industry.Programed by[#66ccff]WinterUnderTheSnow[] on 2020.10.28.
+  description: "Multi-block factory.Provides more steel.\n Needs to enter water.\n Needs to provide refined carbon and refined iron.\n This is the [red]first [orange]multi-block factory[]of THC Industry.Programed by[#66ccff]WinterUnderTheSnow[] on 2020.10.28."
```

Comment:
_SUPER SECRET : YOU CAN SAVE 2 BYTES OF DISK SPACE WHEN REMOVING AN `" "`_

Reply:
emm... actually not placing "" sometime can cause problems
when using with [] or {} typo, i guess.